app/controllers/manifests_controller.rb

```diff
@@ -1,12 +1,8 @@
 # frozen_string_literal: true
 
 class ManifestsController < ActionController::Base # rubocop:disable Rails/ApplicationController
-  # Prevent `active_model_serializer`'s `ActionController::Serialization` from calling `current_user`
-  # and thus re-issuing session cookies
-  serialization_scope nil
-
   def show
     expires_in 3.minutes, public: true
-    render json: InstancePresenter.new, serializer: ManifestSerializer, root: 'instance'
+    render json: InstancePresenter.new, serializer: ManifestSerializer, root: 'instance', public: true
```

Comment:
just noticed this is `render` instead of `render_with_cache` I'm not sure if this should be cached or not?